pkg/cloudprovider/cloud.go

```diff
@@ -23,6 +23,7 @@ import (
 
 	"k8s.io/api/core/v1"
 	"k8s.io/apimachinery/pkg/types"
+	"k8s.io/apiserver/pkg/storage/value/encrypt/envelope"
```

Comment:
I think cloud provider shouldn't import apiserver. Re-declare the interface here.